.github/workflows/aws-eks-test.yml

```diff
@@ -42,8 +42,13 @@ jobs:
           aws eks update-kubeconfig --region ${{ secrets.GHA_SVC_ACC_AWS_REGION }} --name ${{ secrets.GHA_SVC_ACC_EKS_CLUSTER_NAME }}
 
       - name: install acorn
+        # If a previous run left Acorn installed, uninstall it
         run: |
-          acorn install --image ghcr.io/acorn-io/acorn:main
+          if acorn check; then
+            acorn uninstall -af
+          fi
+        run: acorn install --image ghcr.io/acorn-io/acorn:main
```

Comment:
Is it valid to have more than one `run` entry here?

Reply:
Fixed!